chromium_src/BUILD.gn

```diff
@@ -401,6 +401,8 @@ source_set("chrome_spellchecker") {
 
   if (enable_builtin_spellchecker) {
     sources += [
+      "//chrome/browser/profiles/profile.cc",
```

Comment:
we don't support Profiles as they don't map to any analogous concept in Electron - we'll want to patch this out of wherever it's causing an issue, not pull it in